code/modules/reagents/reagent_dispenser.dm

```diff
@@ -262,6 +262,13 @@
 	amount_per_transfer_from_this = 50
 	reagents_to_add = list(/decl/reagent/luminol = 1000)
 
+/obj/structure/reagent_dispensers/peppertank/spacecleaner
+	name = "space cleaner dispenser"
```

Comment:
```suggestion
	name = "cleaner dispenser"
```
"space cleaner" is kind of a old, jokey name from when most things had "space" as a prefix. (e.g. "space milk", "space cola".) i personally don't think it should be added to new objects, if avoidable.